src/docs/streams/quickstart.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: docs.hbs
+title: Streams Introduction
+---
+
+---
+layout: docs.hbs
+title: Streams Introduction
+---
+
+# Quick Start Guide
+A stream usually begins at a source, so this is also how we start an Akka Stream. Before we create one, we import the full complement of streaming tools:
+
+```csharp
+using Akka.Streams;
+using Akka.Streams.Dsl;
+```
+Now we will start with a rather simple source, emitting the integers 1 to 100;
+
+```csharp
+Source<T, NotUsed> source = Source.From(Enumerable.Range(1,100))
+```
+The `Source` type is parametrized with two types: the first one is the type of element that this source emits and the second one may signal that running the source produces some auxiliary value (e.g. a network source may provide information about the bound port or the peer�s address). Where no auxiliary information is produced, the type `NotUsed` is used -- and a simple range of integers surely falls into this category.
+
+Having created this source means that we have a description of how to emit the first 100 natural numbers, but this source is not yet active. In order to get those numbers out we have to run it:
+
+```csharp
+source.RunForeach(i => Console.WriteLine(i.ToString()), materializer)
+```
+This line will complement the source with a consumer function--in this example we simply print out the numbers to the console--and pass this little stream setup to an Actor that runs it. This activation is signaled by having �run� be part of the method name; there are other methods that run Akka Streams, and they all follow this pattern.
+
+You may wonder where the Actor gets created that runs the stream, and you are probably also asking yourself what this materializer means. In order to get this value we first need to create an Actor system:
+```csharp
+using (var system = ActorSystem.Create("system"))
+using (var materializer = system.Materializer())
+```
+There are other ways to create a materializer, e.g. from an `ActorContext` when using streams from within Actors. The `Materializer` is a factory for stream execution engines, it is the thing that makes streams run --you don�t need to worry about any of the details just now apart from that you need one for calling any of the run methods on a `Source`. The materializer is picked up implicitly if it is omitted from the `run` method call arguments, which we will do in the following.
```

Comment:
``` The materializer is picked up implicitly if it is omitted from the `run` method call arguments, which we will do in the following.```

That's not possible for us, we always need to pass the materializer as parameter for run methods